src/inbox/mod_inbox.erl

```diff
@@ -289,6 +317,19 @@ form_field_value(Value) ->
 
 %%%%%%%%%%%%%%%%%%%
 %% Helpers
+get_inbox_unread(Value, Acc, _) when is_integer(Value) ->
```

Comment:
For now it's OK but I think we should note in the backlog that this value should be bound to a stanza reference inside `Acc` (what if someone reuses `Acc` to send another message to different recipient?), in a way that is similar to `mongoose_iq`.

Reply:
So should there be a note in the code left?